session-name-1/activities/activity-folder-1/README.md

```diff
@@ -0,0 +1,17 @@
+# Name: Web/Binary: Session Name: Challenge Name
+
+## Description
+
+Get the flag from [url]http://...:.../[/url].
+
+Score: TODO
```

Comment:
```suggestion
```
Again a variable we might have trouble updating.
Does an external viewer need to know its score? They should only be interested in the challenge itself.
We should keep this mappings somewhere else, along with IP, port, etc.

Reply:
We will keep this information as we discussed in `meta-internal`.
I removed it from the README and I will make the `meta-internal` repo.